modules/develop/pages/produce-data/leader-pinning.adoc

```diff
@@ -0,0 +1,51 @@
+= Leader Pinning
+:page-categories: Clients, Development
+// tag::single-source[]
+:description: Learn about leader pinning and how to configure a preferred partition leader based on availability zones.
+
+Produce requests to write data to Redpanda topics go through the topic partition leader, which syncs messages across its follower replicas. For a Redpanda cluster deployed across multiple availability zones (AZs), leader pinning ensures that a topic's partition leaders are geographically closer to clients. Leader pinning can lower networking costs and help guarantee lower latency by routing produce and consume requests to brokers located in certain AZs.
+
+See also: xref:develop:consume-data/follower-fetching.adoc[Follower fetching] to configure consumers to fetch records from the closest replica and help reduce overall networking costs. 
+
+== Prerequisites
+
+include::shared:partial$enterprise-license.adoc[]
+
+You must xref:manage:rack-awareness.adoc#configure-rack-awareness[configure rack awareness] for the cluster. If the config_ref:enable_rack_awareness,true,properties/cluster-properties[] cluster configuration property is set to `false`, leadership pinning is disabled across the cluster.
+
+== Configure leader pinning
+
+You can use a topic configuration property as well as a cluster configuration property to configure leader pinning.
+
+The following steps assume that you have Redpanda deployed in a multi-AZ setup, and you have configured each broker so that the config_ref:rack,true,properties/broker-properties[] configuration property contains rack IDs corresponding to the AZ IDs.
```

Comment:
Is this a prerequisite?

Reply:
@asimms41 It's part of the prereq to configure rack awareness.